api/Infrastructure/Database/Clients/StageSelectionClient.cs

```diff
@@ -41,6 +41,19 @@ private IEnumerable<StageSelectableRider> GetTeam(int stagenr)
         return team.ToList();
     }
 
+    public IEnumerable<AccountStageResults> GetAccountStageResults(int raceId, int stagenr)
```

Comment:
Kan als oneliner door meteen toList aan het einde te plakken.
maar dit voelt meer als een query die in de andere stijl kan zoals TeamSelectionClient.GetTeam.
Ik ben zelf geen fan van de join in deze versie.
from s in studentList // outer sequence
                      join st in standardList //inner sequence 
                      on s.StandardID equals st.StandardID // key selector 
                      
                      zoiets krijg je dan

Reply:
En dit moet ook wss straks naar een nieuwe client als we meerdere stageresult queries hebben